repos/dir.targets

```diff
@@ -109,6 +109,11 @@
     <AddSourceToNuGetConfig NuGetConfigFile="%(NuGetConfigFiles.Identity)"
                             SourceName="source-built"
                             SourcePath="$(SourceBuiltPackagesPath)" />
+
+    <AddSourceToNuGetConfig NuGetConfigFile="%(NuGetConfigFiles.Identity)"
+                            SourceName="ExtraSources"
+                            SourcePath="$(ExtraRestoreSourcePath)"
+                            Condition="'$(OfflineBuild)' == 'true' AND '$(ExtraRestoreSourcePath)' != ''" />
```

Comment:
Can the property name `ExtraRestoreSourcePath` be improved to indicate it only applies in offline builds? Or should the condition move to where `ExtraRestoreSourcePath` is set?

Reply:
It doesn't need to be offline only.  I removed that and put the OfflineBuild condition on the setting of the property in the arcade repo project.